src/libs/checkForUpdates.js

```diff
@@ -18,6 +18,9 @@ function checkForUpdates(platformSpecificUpdater) {
     setInterval(() => {
         platformSpecificUpdater.update();
     }, UPDATE_INTERVAL);
+
+    // Perform initial check for update
+    platformSpecificUpdater.update();
```

Comment:
This should be done as part of the `platformSpecificUpdater.init` function. I think this would cause a breaking change on the web platform where the page would refresh in an infinite loop.